scripts/tasks/compile.ts

```diff
@@ -19,7 +19,10 @@ export const compile: Task = {
       // To check if the code has been compiled as we need, we check the compiled output of
       // `@storybook/preview`. To check if it has been built for publishing (i.e. `--no-link`),
       // we check if it built types or references source files directly.
-      const contents = await readFile(resolve(codeDir, './lib/preview/dist/runtime.d.ts'), 'utf8');
```

Comment:
To explain:

This is checking if the source of `'./lib/preview/dist/runtime.d.ts'` is this:
```
export * from '../src/index'
```

But of course it isn't! It's going to match the filename:

```
export * from '../src/runtime'
```

Reply:
Also turns out I added a `;` at the end a while ago in the bundle script that I never added here...